src/components/AttachmentPicker/index.native.js

```diff
@@ -28,54 +35,72 @@ const documentPickerOptions = {
 };
 
 /**
- * Launch the DocumentPicker. Results are in same format as ImagePicker, so we can pass the repsonse to the
- * callback as is.
+ * Inform the users when they need to grant camera access and guide them to settings
+ */
+function showPermissionsAlert() {
+    Alert.alert(
+        'ExpensifyCash does not have access to your camera. To enable access, tap Settings and turn on Camera.',
```

Comment:
We should use 'Expensify.cash' here. Also, I think we may be able to remove this entirely? I don't see this message on physical devices, but instead seeing this:

![permissionCamera](https://user-images.githubusercontent.com/10736861/113007418-1cc47800-916e-11eb-9088-e88359fda03b.png)

Reply:
Okay, never-mind. This is built into the library, I'll raise a new issue for this problem.

Reply:
@Julesssss 
On `iOS` I do see this error if I don't grant permission:
![image](https://user-images.githubusercontent.com/12156624/113016170-e6472700-9186-11eb-92ba-82f6798eeb0d.png)

I'll update the text

Now that I've checked this is probably related: https://github.com/react-native-image-picker/react-native-image-picker#android - we have declared `android.permission.CAMERA` in the andorid manifest and so we'll need to ask for Permissions when the camera option is selected. Did this work on Android before? It might be something that was handled internally previously when `RNImagePicker.showImagePicker` was used

Reply:
This is what I see for Android (no modifications to current code):

#### The app asks for permissions when the Camera option is selected
![image](https://user-images.githubusercontent.com/12156624/113026592-eac50d00-9191-11eb-8278-73a68dc9df1c.png)

#### Selecting deny shows me this message on android:
![image](https://user-images.githubusercontent.com/12156624/113026896-35df2000-9192-11eb-9246-077be82a7989.png)

---

Note: with this PR we can upgrade to v3 of this library. Maybe this is handled differently there